monai/transforms/utils_pytorch_numpy_unification.py

```diff
@@ -112,7 +114,7 @@ def percentile(x: NdarrayOrTensor, q, dim: Optional[int] = None) -> Union[Ndarra
     else:
         q = torch.tensor(q, device=x.device)
         if hasattr(torch, "quantile"):  # `quantile` is new in torch 1.7.0
-            result = torch.quantile(x, q / 100.0, dim=dim)
+            result = torch.quantile(x, q / 100.0, dim=dim, **kwargs)
```

Comment:
`np.percentile` also takes some additional params, perhaps we try to support both, like https://github.com/Project-MONAI/MONAI/blob/3c6ee694d5e7fd6d9da2c65ed1b83030488da064/monai/transforms/utils_pytorch_numpy_unification.py#L301-L305

Reply:
Hi @wyli ,

Thanks for your suggestion.
In this case, I found that `numpy` has more args than `tensor`, so I added `keepdim` to the function and use `kwargs` for numpy array.

Thanks.